src/views/Accessibility/AccessibiltyRequest/Create/index.tsx

```diff
@@ -53,13 +53,10 @@ const Create = () => {
           intakeID: values.intakeId
         }
       }
-    }).then(() => {
-      showMessageOnNextPage(
-        t('newRequestForm.confirmation', {
-          requestName: values.requestName
-        })
-      );
-      history.push('/');
+    }).then(res => {
+      const uuid = res.data.createAccessibilityRequest.accessibilityRequest.id;
```

Comment:
This is going to cause an issue if the response has errors on it, I think. Can you maybe do an if statement to check `res.errors`? There are a few other examples of that in the codebase.

Reply:
Good catch, I added some defensive code.

```
if (!response.errors) {
   const uuid =
     response.data.createAccessibilityRequest.accessibilityRequest.id;
   showMessageOnNextPage(t('newRequestForm.confirmation'));
   history.push(`/508/requests/${uuid}`);
 }
```